crowbar_framework/app/models/node_object.rb

```diff
@@ -1451,6 +1451,26 @@ def disk_release!(device, owner)
     disk_release(device, owner) and save
   end
 
+  def verify_claimed_disks
+    unreferenced_disks = Array.new
+    crowbar_wall[:claimed_disks].each do |disk, _claim|
+      path, device_name = disk.split("/")[-2..-1]
```

Comment:
could you add the explanation from @rhafer to the code as a comment? would be helpful for the next one who has to look at this code